### PR TITLE
solution

### PR DESCRIPTION
**Pull request by uzlss**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split(" ")
+    if command[1] != command[2]:
```
Comment (Grayder0152) [inline app/main.py:3]: check that command lenght == 3

File: app/main.py [hunk 2]
```diff
@@ -1 +1,6 @@
...
+        with open(command[1], "r") as file_in,\
+                open(command[2], "w") as file_out:
```
Comment (Grayder0152) [inline app/main.py:5]: use `()` instead of `\`

**Review by Grayder0152 — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 3]
```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split(" ")
```
Comment (ArsenPidhoretskyi) [inline app/main.py:2]: space is default in split
```suggestion
    command = command.split()
```

File: app/main.py [hunk 4]
```diff
@@ -1 +1,6 @@
...
+    if len(command) == 3 and command[1] != command[2]:
```
Comment (ArsenPidhoretskyi) [inline app/main.py:3]: Also check that command is really cp

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]
.

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]
.

**Review by ArsenPidhoretskyi — APPROVED** [review]
Well done!

**Review by ArsenPidhoretskyi — APPROVED** [review]

**Review by ArsenPidhoretskyi — APPROVED** [review]

**Review by ArsenPidhoretskyi — APPROVED** [review]

**Pull request closed without merging**